### ISSUE TITLE
[Low Priority] (My) 4824 will reset during a redumper dump, but not a DiC one.

### ISSUE THREAD

**Issue by HeroponRikiBestest** [superg/redumper#165]
Not expecting this to be fixed, mainly just posting it for documentation purposes. My 4824 tends to reset in the middle of redumper dumps of larger CDs; if a CD is completely full, it tends to make it about halfway through before resetting. This doesn't result in bad or failed dumps, but just means i have to wait for my drive to hang, reset itself, and then continue the dump. This is consistently reproducible across every version of redumper, and I have never found any way to avoid it. On the latest release of DiC, this does not occur, and the drive will never reset during a dump. Unfortunately, it's difficult to see if this changed with a more recent version, as older versions of DiC often have unrelated issues for me that make it difficult to test. I do remember older versions of DiC still having the drive reset midway through, though. As mentioned, I don't expect this to be fixed, due to the fact that it doesn't produce bad dumps, is specific to an already less recommended drive, and also because said drive seems to have variance among different users. That said, if anyone does want to try and fix this, let me know what additional information needs to be provided.

**Comment by superg** [comment]
Does that happen for every large CD, e.g. any particular kind of disc, multisession etc?
Can't think of anything that might affect it except some timing configurations.

**Comment by HeroponRikiBestest** [comment]
Yep. As mentioned, I honestly wouldn't worry about it right now, since yeah, I can't think of what it would be either.
